### ISSUE TITLE
Switch a11y

### ISSUE THREAD

**Issue by Heydon** [iTwin/stratakit#1236]
## 1. increase the Switch contrast

Currently, there's a serious Non-text Contrast violation. The handle of the Switch is not visible at all. 

<img alt="invisible white switch anatomy" width="333" height="149" alt="Image" src="https://github.com/user-attachments/assets/19712ed9-c01c-47f5-80e2-b376639f449a" />
